### DISCUSSION TITLE
Για το μάθημα της Πέμπτης 12/01/23

### DISCUSSION THREAD

**Discussion by akimo13** [upatras-hci/iv#73] [Q&A] [answered]
Καλησπέρα κ. Χωριανόπουλε, καλή χρονιά. Σήμερα, όλοι όσοι συμμετέχουμε στο μάθημα, συνδεθήκαμε στο ZOOM για να παρακολουθήσουμε τη σημερινή διάλεξη. Παρ' όλ' αυτά μέχρι και αυτή την ώρα 19:30 φαίνεται πως δεν μπορούμε να συνδεθούμε.
Συνέβη κάτι ή υπήρξε κάποια παρανόηση στο πότε θα κάνουμε μάθημα;

**Comment by e-liouta** [comment, marked as answer]
![meeting 12 01 chorianopoulos](https://user-images.githubusercontent.com/17378923/212139058-60082334-316c-4bc0-bf63-3b5760cf340f.PNG)

**Reply by epidrome** [reply]
πράγματι, δεν κάνετε λάθος, ξέχασα να μπω στο μάθημα, συγνώμη, δεν έχω κάποια δικαιολογία! 

ας διορθώσουμε αυτό το λάθος άμεσα:

μπορείτε να ανοίξετε μια ψηφοφορία για ώρες που σας βολεύουν την επόμενη εβδομάδα;

άνοιξα ήδη ένα θέμα για τις τελικές παρουσιάσεις #74 όπου δηλώνετε ποια εβδομάδα-μάθημα σας βολεύει

**Reply by e-liouta** [reply]
Μήπως θέλετε να κάνετε εσείς ένα poll με απογευματινές ώρες που να βολεύουν και εσάς και να ψηφίσουμε εμείς;
